src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+
+@mixin iui-skip-to-content {
+  @include iui-blur;
+  @include iui-focus($offset: 2px, $thickness: 2px);
+
+  border-radius: 99999px;
+  color: t(iui-color-foreground-accessory);
```

Comment:
I don't see any background-color, I think we might want to add one because the contrast ratio is inaccessible in light theme

Reply:
`@mixin iui-blur` has a `background-color` of black w/ some level of opacity based on if the blur is supported or not.

Reply:
ok I see. My comment is still valid though, as the contrast ratio needs to be higher than 4.5
![image](https://user-images.githubusercontent.com/9084735/152861636-bd28498f-6ec9-4ec7-8396-370e2d755e84.png)

Reply:
Fixed